### PR TITLE
Pull request for the testfile updateListEntries_ms_test.php issue#15699

### PR DESCRIPTION
**Pull request by a22eriwi**
Imlemented the testfile updateListEntries_ms_test.php to test the ms file updateListEntries_ms.php

### CODE REVIEW

**Review by b21isabe — CHANGES_REQUESTED** [review]
The test fails because the expected output was not gathered from a fresh db install, 
This section of the output is not present when on a fresh install:
```
{"comments":"TOP","entryname":"New Test","feedbackenabled":0,"feedbackquestion":null,"gradesys":0,"grptype":null,"highscoremode":0,"kind":3,"link":"UNK","moment":null,"visible":0},{"comments":"TOP","entryname":"New Code","feedbackenabled":0,"feedbackquestion":null,"gradesys":0,"grptype":null,"highscoremode":0,"kind":2,"link":"9012","moment":null,"visible":0},{"comments":"TOP","entryname":"New Group","feedbackenabled":0,"feedbackquestion":null,"gradesys":0,"grptype":null,"highscoremode":0,"kind":6,"link":"9013","moment":null,"visible":0}
```

Please reinstall your database and then gather the expected output again.

It has also been decided that we should be using the Testing-Course for the tests, so this should also be changed.
Courseid: 1885 coursevers: 1337

Otherwise the test looks good. The updated listEntry can be seen in the output.

File: DuggaSys/tests/microservices/sectionedService/updateListEntries_ms_test.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        'query-before-test-1' => "INSERT INTO listentries (lid, cid, entryname, link, vers, kind, pos, creator, visible, comments, moment, gradesystem, highscoremode, groupKind, feedbackenabled, feedbackquestion)   
+                                  VALUES(9999,1,'Test',8888,45656,0,20,22, 0,'Comment NOT updated',7777,0,0,'NOT updated',0,'Question NOT updated');",
```
Comment (b21isabe) [inline DuggaSys/tests/microservices/sectionedService/updateListEntries_ms_test.php:12]: You forgot to update cid/vers in the query-before-test.

**Review by b21isabe — CHANGES_REQUESTED** [review]
The updated listEntry can't be seen in the value output or expected output, see highlights for fix.

**Comment by a22eriwi** [comment]
I hope everything is fine now! @b21isabe

**Comment by b21isabe** [comment]
Looks good! merging

**Pull request merged**